packages/core/src/Stores/client-store.js

```diff
@@ -1635,7 +1635,7 @@ export default class ClientStore extends BaseStore {
         if (this.is_logged_in) {
             this.getWalletMigrationState();
 
-            await WS.mt5LoginList().then(this.responseMt5LoginList);
+            await WS.authorized.mt5LoginList().then(this.responseMt5LoginList);
```

Comment:
will this have any impact on the user? From what yashim has explained to me, this will slow the call a little since it will check for auth first before calling mt5_login_list, but will this impact any other major flows?

Reply:
we are already checking `this.is_logged_in` then why need `authorized` again?

Reply:
because for certain PA accounts, (idk why this happens) it will call this specific `mt5_login_list` endpoint before we authorize during login. Currently when we login, one websocket is created, and then a second is created, and this `mt5_login_list` gets called in the second websocket, before authorize. So we need to use `WS.authorized` to make sure it waits for authorized call before calling `mt5_login_list`